ci-operator/step-registry/upi/libvirt/conf/upi-libvirt-conf-commands.sh

```diff
@@ -42,4 +42,4 @@ fips: false
 EOF
 
 yq-v4 eval ".pullSecret = load_str(\"${CLUSTER_PROFILE_DIR}/pull-secret\")" -i "${CONFIG}"
-yq-v4 eval ".sshKey = load_str(\"${CLUSTER_PROFILE_DIR}/ssh-publickey\")" -i "${CONFIG}"
\ No newline at end of file
+yq-v4 eval ".sshKey = load_str(\"${CLUSTER_PROFILE_DIR}/ssh-publickey\")" -i "${CONFIG}"
```

Comment:
This is Dominik's conf file, so I just want to make sure that our own workflow is using:
`upi-conf-libvirt` and not this one.

Reply:
According to the workflow we have it right, so this change is probably just an accidentally opened/saved file.